src/eos-crash-metrics.c

```diff
@@ -41,6 +70,18 @@ report_crash (const char *binary,
   g_variant_dict_insert_value (&dict, "ostree_commit", g_variant_new_string (ostree_commit));
   g_variant_dict_insert_value (&dict, "ostree_url", g_variant_new_string (ostree_url));
 
+  if (info != NULL)
+    {
+      g_variant_dict_insert_value (&dict, "app_ref",
+                                   g_variant_new_string (flatpak_ref_format_ref (FLATPAK_REF (info->app))));
```

Comment:
`flatpak_ref_format_ref()` returns a new string, so you should use `g_variant_new_take_string()`

Reply:
Oh, thanks good hint, fixed.